code_of_laba-5.py

```diff
@@ -0,0 +1,82 @@
+class Furniture:
+    def __init__(self, name = "", coordinates =[], size = []):
+        self.name = name
+        self.coordinates = coordinates
+        self.size = size
+
+    def __str__(self):
+        return f" Об*єкт {self.name} має координати {self.coordinates} та розмір {self.size} "
+    
+
+class Room:
+    def __init__(self, size_room = []):
+        self.size_room = size_room
+        self.furniture_pieces_in_room = []

```

Comment:
```suggestion
class Room:
    def __init__(self, size_room=()):
        self.size_room = size_room
        self.furniture_pieces_in_room = []
```
Because assigning a list to an argument in the __init__ method arguments list causes the argument to be mutable